### PR TITLE
Operations workflow for customizing builder images demo

### PR DESCRIPTION
**Pull request by themoosman**
New pull request for Operations Workflow for Customizing Builder Images

### CODE REVIEW

**Comment by etsauer** [comment]
cc @sabre1041 @trevorquinn

File: demos/operations-workflow-for-customizing-builder-images/README.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+```
+
+
+Launch a web browser and navigate to the OSE login page.  Login as `dev` and monitor the image build and deploy.  Finally show how the application is using the `ops-custom-eap6` image.
+As the `dev` user open the `ops-custom-eap6-proj` and demonstrate how the `dev` user cannot scale the application (security).
+
+Launch a web browser and navigate to the OSE login page.  Login as `tsar` and show how this user has full access to both projects.
+
+
+## Presenter Notes
```
Comment (etsauer) [inline demos/operations-workflow-for-customizing-builder-images/README.md]: You've got presenter notes twice

File: demos/operations-workflow-for-customizing-builder-images/README.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+None
+
+### Config Files
+
+None
+
+### External Source Code Repositories
+
+* [Ticket Monster](https://github.com/jboss-developer/ticket-monster) -  Public repository for the Ticket Monster application, used to generate war artifact.  
+* [Ops Repo - EAP6 Customization](https://github.com/themoosman/ops-custom-eap6) -  Public repository that holds configuration information (Infrastructure as Code) used to generate (in our case) a custom EAP6 image.
```
Comment (etsauer) [inline demos/operations-workflow-for-customizing-builder-images/README.md]: We would like to migrate these peripheral repos into the rhtconsulting org before we publish it.

Comment (themoosman) [inline demos/operations-workflow-for-customizing-builder-images/README.md]: @etsauer Do you mean the rhc-ose GitHub or the internal GitLab instance?

Comment (etsauer) [inline demos/operations-workflow-for-customizing-builder-images/README.md]: @themoosman I mean changing the namespace of https://github.com/themoosman/ops-custom-eap6 to https://github.com/rhtconsulting/ops-custom-eap6

**Comment by frankk74** [comment]
Could i get access to themoosman:operations-workflow-for-customizing-builder-images-demo

Thx

Frank

**Comment by themoosman** [comment]
@frankk74 I just double checked and you have push access.

**Comment by frankk74** [comment]
Thx .. I missed that!

**Comment by themoosman** [comment]
@etsauer Bump

**Comment by etsauer** [comment]
@themoosman I think I'm good with this, but I'd like @sabre1041 to weigh in as well.

File: demos/operations-workflow-for-customizing-builder-images/README.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+* [Setup Instructions](#setup-instructions)
+* [Presenter Notes](#presenter-notes)
+	* [Main Demo Steps](#main-demo-steps)
+* [Resources](#resources)
+
+
+
+##Overview
+
+The demo aims to show how the Ops group can customize an external image (Red Hat EAP6) and then have that image used by development.  The following diagram shows that process at a high level.  The demo
```
Comment (sabre1041) [inline demos/operations-workflow-for-customizing-builder-images/README.md]: I would specify it as Red Hat JBoss Enterprise Application Platform (EAP) 6

File: demos/operations-workflow-for-customizing-builder-images/README.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+* [Ticket Monster](https://github.com/jboss-developer/ticket-monster) -  Public repository for the Ticket Monster application, used to generate war artifact.  
+* [Ops Repo - EAP6 Customization](https://github.com/rhtconsulting/ops-custom-eap6) -  Public repository that holds configuration information (Infrastructure as Code) used to generate (in our case) a custom EAP6 image.
+* [Ticket Monster s2i Image Build](https://github.com/rhtconsulting/ticket-monster-ose-s2i-build) - Public repository that holds the scripts (s2i) used to consume development artifacts (war) and create a docker image.
+
+
+## Setup Instructions
+
+There are two general requirement for this demo.
+1. The presenter should have an OpenShift Enterprise 3 environment available and the OpenShift Command Line Tools installed on their machine
+2. A Jenkins server (or other artifact repository) is required to host the development artifact (war)
```
Comment (sabre1041) [inline demos/operations-workflow-for-customizing-builder-images/README.md:59]: The numbered list is not formatting correctly

File: demos/operations-workflow-for-customizing-builder-images/README.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+
+## Presenter Notes
+
+The following steps are to be used to demonstrate how Dev code and Ops code are merged in the build pipeline on OpenShfit V3.
+
+
+### Main Demo Steps
+
+In the demo 3 separate users will be need.
```
Comment (sabre1041) [inline demos/operations-workflow-for-customizing-builder-images/README.md:68]: Does this need to be completed on the OpenShift master?

File: demos/operations-workflow-for-customizing-builder-images/README.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+```
+
+Log in as the `ops` user
+```bash
+oc login -u ops
+```
+
+Create the Ops application
+```bash
+oc new-app registry.access.redhat.com/jboss-eap-6/eap-openshift~http://github.com/rhtconsulting/ops-custom-eap6.git --name=ops-custom-eap6
```
Comment (sabre1041) [inline demos/operations-workflow-for-customizing-builder-images/README.md]: Which project is this application being created in?

**Comment by sabre1041** [comment]
Looks very good. Made a couple minor suggestions. Is is also possible to have an .odp version of the presentation available for those who would want to adjust and do not have pptx?

**Comment by themoosman** [comment]
@sabre1041 I made the changes per your suggestions.  Nice catch on the missing --namespace parameter.

**Comment by etsauer** [comment]
@themoosman looks like you still have that ordered list misformatted under "Setup Instructions"

**Comment by etsauer** [comment]
lgtm. going to merge.

**Pull request merged**